### PR TITLE
Fix for #26 and other minor updates

### PR DESCRIPTION
**Pull request by arghdos**
-correct temperature range column
-check all species in reaction exist
-fix for non-unity default-efficiency (should solve #26)
-duplicate troe warning for chemically activated reactions

### CODE REVIEW

File: pyjac/core/mech_interpret.py [hunk 1]
```diff
@@ -955,6 +957,21 @@ def read_mech_ct(filename=None, gas=None):
     # Cantera internally uses joules/kmol for activation energy
     E_fac = act_energy_fact['joules/kmole']
 
+    def handle_effiencies(reac, ct_rxn):
+        # See if single species acts as third body
```
Comment (kyleniemeyer) [inline pyjac/core/mech_interpret.py]: could you add a proper docstring here, and document what each argument is?

**Comment by kyleniemeyer** [comment]
I think this looks good—did you double check that it doesn't mess anything up that previously works?

File: pyjac/core/mech_interpret.py (hunk 1 above)
Comment (arghdos) [reply pyjac/core/mech_interpret.py]: 90c9a62

Comment (kyleniemeyer) [reply pyjac/core/mech_interpret.py]: 👍

**Comment by arghdos** [comment]
@kyleniemeyer I'll run a test on GRI-3.0 real quick to make sure it didn't break anything

**Comment by arghdos** [comment]
@michael-a-hansen is there an option to specify a `default_efficiency` in a .cti file?  I can't seem to find it.  I'm trying to figure out if this problem shows up in any mechanism we've tested (if not, a simple check of GRI-mech should tell me whether or not I broke anything)

**Comment by michael-a-hansen** [comment]
I can't find a default efficiency specification in any CTI files. It looks like the only case a default efficiency is zero is when the species is specified directly, as in `H + O2 (+ AR) <=> HO2 (+ AR)`. It certainly looks like the default efficiency will always be 1 or 0, but I suppose it's good to either catch that and/or match cantera behavior anyway.

**Comment by arghdos** [comment]
>but I suppose it's good to either catch that and/or match cantera behavior anyway

Agreed! I don't think the fix here should change anything for 99.9% of mechanisms, but Cantera _does_ allow (or at least, doesn't reject) setting the third-body efficiency of a single species third-body (as in the test case you gave).

**Comment by arghdos** [comment]
Testing on the first GRI-mech PaSR database gave:

```
>>> np.mean(err['err_jac_thr']) 
0.010123234852160436
>>> np.mean(err['err_jac_thr']) / err['err_jac_thr'].size
2.0206057589142586e-07
>>> np.max(err['err_jac_thr_max'])
0.0067045055745845851
```
which is ~ the numbers in Table 3 of the paper, so i think we should be good here

**Comment by arghdos** [comment]
@kyleniemeyer I've updated the changelog as well.  This might be a good time to do a new release as well (bumping up to 1.0.5.b1).  If we don't have any more immediate concerns, I think we should merge this in

**Comment by kyleniemeyer** [comment]
OK, seems good. Agreed on bumping the version number.

**Pull request merged**